gui/wxpython/gui_core/preferences.py

```diff
@@ -1245,6 +1272,54 @@ def _createCmdPage(self, notebook):
 
         gridSizer.Add(verbosity, pos=(row, 1), flag=wx.ALIGN_RIGHT)
 
+        row += 1
+        # nprocs
+        gridSizer.Add(
+            StaticText(
+                parent=panel,
+                id=wx.ID_ANY,
+                label=_("Number of threads for parallel computing:"),
```

Comment:
I wonder if we should add "...computing (supported tools only): ". Or "Number of cores to be used by parallelized tools".
Similarly, the memory handling is implemented only in certain tools, so maybe "Maximum memory in MB to be used (supported tools only)". But maybe you can come up with a better wording... I just don't want this to be misunderstood.

Reply:
Fixed with https://github.com/OSGeo/grass/pull/3307/commits/97735efa011f9eed5b9105e1cc14ff4a2a3192c6.